README.md

```diff
@@ -1 +1,50 @@
 # HangOps Code of Conduct
+
+To continue growing a safe, welcoming, and fair community, all HangOps members are empowered to call out and attempt to resolve disrespectful behavior, and admins are empowered to enforce the Code of Conduct.
+
+## The Short Version
+
+Be respectful of other people; respectfully call out anyone behaving inappropriately, ask them to stop, and direct them to this Code of Conduct; and if you need help resolving an issue, contact the admins. If you are unable to follow our Community Code of Conduct you will be removed from HangOps.
+
+## The Long Version
+
+### Respect
+
+HangOps is a community of people who care about running production systems reliably. We have diverse backgrounds, skills, talents, experiences, cultures, and opinions. We are an inclusive environment and we treat all individuals respectfully.
+
+We value respectful behavior above individual opinions.
+
+Respectful behavior includes:
+
+* Be considerate, kind, constructive, and helpful.
+* Avoid demeaning, discriminatory, harassing, hateful, or physically threatening behavior, speech, and imagery.
+* If you’re not sure, ask someone instead of assuming. No, really. Just ask the admins.
+* Model respectful behaviour, but also address disrespect when you see it. One of the best ways to create a respectful community is to speak up when you see disrespectful behavior—and not just rely on admins to act!
+
+### Resolve Peacefully
+
+If you see someone behaving disrespectfully, we urge you to respectfully dissuade them from such behavior.
```

Comment:
Example 1) I expect would be a case-by-case with some initiative by the original victim to indicate that they are now comfortable with the prior abuser's presence in hangops

Example 2) I may not follow the nuance of, but my sense is that if one hangops user abuses another, the abuser needs to go. I don't see how that would be different based on being an existing community member vs a new one. But like I said, maybe I'm missing some nuance in your scenario.